app/views/users/registrations/edit.html.erb

```diff
@@ -1,32 +1,54 @@
+<head>
```

Comment:
This should be unnecessary since we are including our application css through rails. We shouldn't need to manually add the css to the html

Reply:
You might wanna take a look at the "assets pipeline" in rails. It should talk about how it gets the css and js into the application.

Reply:
Okay I push it up with using application.css to do the mapping of the CSS and I hope I did it correctly and change the tag ```id``` to ```class``` instead.